frp, frpc, frps: add page (#12945)
tldr-pages/tldr: pages/common/frpc.md

# frpc

> Connect to a `frps` server to start proxying connections on the current host.
> Part of `frp`.
> More information: <https://github.com/fatedier/frp>.

- Start the service, using the default config file (assumed to be `frps.ini` in the current directory):

`frpc`

- Start the service, using the newer TOML config file (`frps.toml` instead of `frps.ini`) in the current directory:

`frpc --config ./frps.toml`

- Start the service, using a specified config file:

> `frpc --config {{path/to/file}}`

Author: Should I treat the same with `{{-v|--version}}` ?

> `frpc --config {{path/to/file}}`

Reviewer: IG it's fine to have the same syntax for those examples.

> - Verify that the configuration file is valid:

Reviewer 2: ```suggestion
- Check if the configuration file is valid:
```

`frpc verify --config {{path/to/file}}`

> - Generate `frpc` command autocompletion setup script for Bash, Fish, PowerShell, or Zsh (printed to `stdout`):

Reviewer 2: ```suggestion
- Print autocompletion setup script for Bash, Fish, PowerShell, or Zsh:
```

What about this?

`frpc completion {{bash|fish|powershell|zsh}}`

- Display version:

> `frpc --version`

Reviewer 3: ```suggestion
`frpc {{-v|--version}}`
```
